rosa_hcp/terraform/rosa-hcp-creating-a-cluster-quickly-terraform.adoc

```diff
@@ -0,0 +1,33 @@
+:_content-type: ASSEMBLY
+[id="rosa-hcp-creating-a-cluster-quickly-terraform"]
+= Creating a default ROSA cluster using Terraform
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-creating-a-cluster-quickly-terraform
+
+toc::[]
+
+Create a {product-title} (ROSA) cluster quickly by using a Terraform cluster template that is configured with the default cluster options.
+
+The cluster creation process described below uses a Terraform configuration that prepares a {hcp-title} cluster with the following resources:
+
+* An OIDC provider with a managed `oidc-config`
```

Comment:
```suggestion
* An OIDC provider with a managed `oidc-config` configuration
```

Reply:
It seems repetitive to say config configuration but I wasn't sure if that was the noun that is needed to follow the config for clarity.